pages/initiatives/gsoc/gsoc2022ideas.md

```diff
@@ -105,4 +105,48 @@ list? Great, please submit it!
 * Jannik Hollenbach - OWASP Juice Shop Core Team
 * Timo Pagel - OWASP Juice Shop Core Team
 
+### [OWASP ZAP](https://zaproxy.org/)
+
+ZAP is the world’s most widely used web scanner. Previous GSoC contributors have added key features and are all listed in the [ZAP Student Hall of Fame](https://www.zaproxy.org/student-hall-of-fame/).
+
+To get started with ZAP contributions see the [ZAP Contributing Guide](https://www.zaproxy.org/docs/contribute/). We expect GSoC contributors who apply to work on ZAP to have had at least one PR merged into one of the ZAP repos.
+
+##### Import Postman API Definitions
+
+![Possible for "Medium" GSoC 2022 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
+![Preferred for "Large" GSoC 2022 project](https://img.shields.io/badge/large%20size%20(~350h)-preferred-green)
+
+Import Postman API definitions into ZAP as per [#6960](https://github.com/zaproxy/zaproxy/issues/6960)
+
+##### Import PCAP/PCAPNG Files
+
+![Possible for "Medium" GSoC 2022 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
+![Preferred for "Large" GSoC 2022 project](https://img.shields.io/badge/large%20size%20(~350h)-preferred-green)
```

Comment:
Do we really think this will take 8 to 10 weeks?

Reply:
TBH I just applied that to all of them as I wasnt sure. Do you think it will be quicker? I dont know the details well enough to judge but happy for us to change it...

Reply:
Well I had a quick look a libraries etc and to me it "should" be quicker :wink: